src/audio-worklet/migration/worklet-recorder/index.njk

```diff
@@ -23,7 +23,13 @@ to_root_dir: ../../../
 </style>
 
 <h1>AudioWorklet Recorder</h1>
-<p>Demonstrates a simple recording app using Audio Worklet</p>
+<p>This is a simple recording app that uses the AudioWorklet API.
+The maximum recording time is 5 minutes. After reaching the maximum
```

Comment:
The recommended style is 2-space indentation at L27 and below until `</p>`.

You can merge this and fix it with a new PR, or you can do it in here. It's up to you!

Reply:
Thanks for pointing it out! I think I will keep it in mind and do it in the next PR since this PR is moving the commits into [main](https://github.com/GoogleChromeLabs/web-audio-samples) from [exploration-sprint-2023](https://github.com/GoogleChromeLabs/web-audio-samples/tree/exploration-sprint-2023)